pom.xml

```diff
@@ -13,23 +13,24 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>commons-codec</groupId>
       <artifactId>commons-codec</artifactId>
-      <version>1.11</version>
+      <version>1.13</version>
     </dependency>
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
-      <version>3.5</version>
+      <version>3.8.1</version>
```

Comment:
As of the time this PR was filed, there exists a `3.9` version, but it compiles to Java 1.8. `3.8.1` on the other hand compiles to Java 1.7 which is the oldest version we support.